addons/block_code/simple_spawner/simple_spawner.gd

```diff
@@ -99,12 +100,15 @@ func spawn_once():
 	var scene: PackedScene = scenes.pick_random()
 	var spawned = scene.instantiate()
 	_spawned_scenes.push_back(spawned)
-	match spawn_parent:
-		SpawnParent.THIS:
-			add_child(spawned)
-		SpawnParent.SCENE:
-			get_tree().current_scene.add_child(spawned)
-			spawned.position = global_position
+	# Rotate the spawned scene according to the SimpleSpawner:
+	if spawned is RigidBody2D:
+		spawned.linear_velocity = spawned.linear_velocity.rotated(global_rotation)
+		spawned.constant_force = spawned.constant_force.rotated(global_rotation)
+	elif spawned is Node2D:
+		spawned.rotate(global_rotation)
+	# Add the spawned instance to the current scene:
+	get_tree().current_scene.add_child(spawned)
```

Comment:
Sorry to be jumping in about unrelated things (it's just we never really had the chance to review this properly :b), but _while we're here_, is there a good reason this is dumping the spawned node in the root of the scene instead of doing `add_sibling(spawned)`? I can see how it was easier to express in the options list we had before, but at this point it's rather surprising behaviour.

Reply:
Sorry, I didn't see your comment until now @dylanmccall.

I just tried the "siblings" alternative: https://github.com/endlessm/godot-block-coding/tree/simple-spawner-rotation-alt2

But for the space ship demo, it makes the blasters rotate with the ship:
[Grabación de pantalla desde 2024-10-30 10-58-13.webm](https://github.com/user-attachments/assets/a97b0f06-a2e6-4b66-8341-28413343c4f1)


This is because the SimpleSpawner is a child of the ship that's being rotated with keystrokes.

Reply:
Oooh right, that makes sense :) Thanks for reminding me!